### ISSUE TITLE
queation on replicating your results

### ISSUE THREAD

**Issue by chenbin234** [Fritschek/ProductAE#1]
Hi Fritschek, thanks for sharing the code, one questions here, I retrained the model using Notebook_main.ipynb but obtained different results (see figure below). Have you made any recent changes?
I'd appreciate your guidance on how to replicate your results, as I'd like to experiment with a larger batch size to see if it improves performance.
![image](https://github.com/user-attachments/assets/58f780ac-833e-4b30-be5e-e0c2ed1cea15)

**Comment by Fritschek** [comment]
I'll check it, had a quick look and I can't remember why I did the sparse loss in the training code.
Internally, I use a different training code, and the NN model seems to work fine (running the test right now).
I do use a bigger hidden size, note that in the paper they use 200 and 250 for encoder and decoder respectively, and not what I did here. I'll update the training code here, once I have a bit more time.
